go.mod

```diff
@@ -1,6 +1,8 @@
 module knative.dev/eventing-rabbitmq
 
-go 1.18
+go 1.21
+
+toolchain go1.21.6
```

Comment:
This should not be here.
--> Brought up https://github.com/knative-extensions/eventing-rabbitmq/pull/1337

/hold